dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/trackedentity/hibernate/TrackedEntityAttribute.hbm.xml

```diff
@@ -69,14 +70,16 @@
 
     <property name="pattern" />
 
-    <property name="textPattern" type="jbTextPattern" column="textpattern" not-null="false"/>
+    <property name="textPattern" type="jbTextPattern" column="textpattern" not-null="false" />
 
     <property name="style" type="jbObjectStyle" column="style" />
 
     <property name="orgunitScope" />
 
     <property name="programScope" />
 
+    <property name="skipSynchronization" column="skipsynchronization" not-null="false" />
```

Comment:
when name == column, you don't need to specify column name.

Reply:
Explicit column name removed.